File: Domain/ObjetoPostReserva.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class ObjetoPostReserva
    {
        public HospedajeFiltro Filtro { get; set; }
        public DatosUsuario Datos { get; set; }
        public int? HospedajeId { get; set; }
    }
}

File: SPA/FrontEnd/src/app/Models/ObetoPostReserva/objeto-post-reserva.ts
import { DatosUsuario } from '../Datos/DatosUsuario/datos-usuario';
import { HospedajeFiltro } from '../HospedajeFiltro/hospedaje-filtro';
export class ObjetoPostReserva {
    filtro: HospedajeFiltro;
    datos: DatosUsuario;
    hospedajeId: number;

    constructor(Filtro: HospedajeFiltro, Datos: DatosUsuario, HospedajeId: number) {
        this.datos = Datos;
        this.filtro = Filtro;
        this.hospedajeId = HospedajeId;
    }
}
